### PR TITLE
[Cherry-pick] PAC: creates pipelinerun templates in openshift namespace

### PR DESCRIPTION
**Pull request by sm43**
Signed-off-by: Shivam Mukhade <smukhade@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: pkg/reconciler/openshift/tektonaddon/pipelinesascode.go [hunk 1]
```diff
@@ -157,7 +157,13 @@ func (r *Reconciler) getManifest(ctx context.Context, ta *v1alpha1.TektonAddon)
 		return nil, err
 	}
 
-	if err := fetchPRTemplates(&pacManifest); err != nil {
+	prTemplates := mf.Manifest{}
+	if err := fetchPRTemplates(&prTemplates); err != nil {
+		return nil, err
+	}
+	var err error
+	prTemplates, err = prTemplates.Transform(mf.InjectNamespace("openshift"))
```
Comment (chmouel) [inline pkg/reconciler/openshift/tektonaddon/pipelinesascode.go:165]: should this be a constant?

**Review by concaf — APPROVED** [review]
/lgtm

**Pull request merged**